src/server/replication_buffer.rs

```diff
@@ -1,377 +1,72 @@
-use std::{
-    io::{Cursor, Write},
-    mem,
-};
-
-use bevy::{prelude::*, ptr::Ptr};
-use bincode::{DefaultOptions, Options};
-use serde::Serialize;
-use varint_rs::VarintWriter;
-
-use crate::{
-    replicon_core::replication_rules::{ReplicationId, ReplicationInfo},
-    server::ClientMapping,
-};
+use std::{io::Cursor, ops::Range};
 
 /// A reusable buffer with replicated data.
-pub(super) struct ReplicationBuffer {
+#[derive(Default)]
+pub(crate) struct ReplicationBuffer {
```

Comment:
This files was completely rewritten, I would suggest to look only at added lines.